scripts/run-unit-tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+count=`ls -1 src/**/*.unit.ts -name 'Prams' -type d 2> /dev/null | wc -l`
```

Comment:
No issue when no tests exist. When I copied a test file in, it still said "write some tests!".

How about: ?
```suggestion
count=`find ./src -name '*.unit.ts' | wc -l`
```